service/src/main/kotlin/fi/espoo/evaka/vasu/VasuController.kt

```diff
@@ -177,6 +178,8 @@ class VasuController(
                     vasu.content.editFollowupEntry(entryId, evakaClock, editedBy, body.text),
                     vasu.basics.childLanguage
                 )
+
+                tx.revokeVasuGuardianHasGivenPermissionToShare(id)
```

Comment:
Are the guardians supposed to give permissions multiple times or is it supposed to be given then the document is "finished"? I'm just asking really.

Reply:
Each time the document is changed it must be given permission to share again